### PR TITLE
feat: add Belarusian translation

### PR DESCRIPTION
**Pull request by StIvan8**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [x] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

Added locales for Belarusian translation.
| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Add Belarusian locale.  |
| 🇨🇳 Chinese | 新增白俄罗斯语支持。 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
Please refer to #21387, and you need to submit PR to [rc-picker](https://github.com/react-component/picker) and [rc-pagination](https://github.com/react-component/pagination) first.

**Comment by yoyo837** [comment]
[rc-picker](https://github.com/react-component/picker): [v2.3.0](https://github.com/react-component/picker/releases/tag/v2.3.0)
[rc-pagination](https://github.com/react-component/pagination): [v3.1.0](https://github.com/react-component/pagination/releases/tag/v3.1.0)

**Comment by afc163** [comment]
Please fill these parts: 

image

**Comment by afc163** [comment]
- [ ] Add test case: https://github.com/ant-design/ant-design/blob/master/components/locale-provider/__tests__/index.test.js
- [ ] Add Locale to documentation
  - [ ] https://github.com/ant-design/ant-design/blob/master/docs/react/i18n.en-US.md
  - [ ] https://github.com/ant-design/ant-design/blob/master/docs/react/i18n.zh-CN.md

**Comment by yoyo837** [comment]
Commits should be clean, seems mixed with other commits.

File: package.json [hunk 1]
```diff
@@ -132,12 +132,12 @@
...
     "rc-pagination": "~3.0.3",
-    "rc-picker": "~2.1.0",
+    "rc-picker": "~2.3.0",
     "rc-progress": "~3.1.0",
     "rc-rate": "~2.8.2",
     "rc-resize-observer": "^0.2.3",
-    "rc-select": "~11.4.0",
-    "rc-slider": "~9.5.2",
+    "rc-select": "~11.3.2",
+    "rc-slider": "~9.5.1",
```
Comment (yoyo837) [inline package.json:140]: Why change this？

Comment (StIvan8) [reply package.json:140]: I use npm i

Comment (StIvan8) [reply package.json:140]: fix it?

Comment (afc163) [reply package.json]: Don't change it.

File: package.json [hunk 2]
```diff
@@ -131,8 +131,8 @@
     "rc-menu": "~8.7.1",
     "rc-motion": "^2.2.0",
     "rc-notification": "~4.5.2",
-    "rc-pagination": "~3.0.3",
-    "rc-picker": "~2.1.0",
+    "rc-pagination": "~3.1.0",
+    "rc-picker": "~2.3.0",
```
Comment (yoyo837) [inline package.json:135]: 发布的同学changelog记得写

https://github.com/react-component/picker/pull/142

https://github.com/react-component/picker/pull/144

https://github.com/react-component/picker/pull/145

**Pull request merged**